src/theme/theme.ts

```diff
@@ -84,6 +84,11 @@ export const theme: HerzUITheme = {
       fontFamily: "body",
       color: "text.0",
     },
+    secondary: {
+      color: "secondary.0",
+      fontSize: 14,
+      lineHeight: 1.45,
+    },
```

Comment:
Is this typography variant defined somewhere in the design system?
I think we should avoid setting colors in the text variants, and the other styles in this variant can be achieved by just using one of the other variants.